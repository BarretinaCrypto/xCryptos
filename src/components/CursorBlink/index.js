import React from 'react';
import { View as Motion } from 'react-native-animatable';

import styles from './style';

const CursorBlink = () => <Motion animation="fadeIn" duration={500} iterationCount="infinite" style={styles.blink} />;

export default CursorBlink;
