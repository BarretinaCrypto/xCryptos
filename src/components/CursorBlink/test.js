import 'react-native';
import React from 'react';
import renderer from 'react-native-test-utils';

import CursorBlink from './index';

jest.mock('TouchableOpacity', () => 'TouchableOpacity');

it('renders by default', () => {
  const tree = renderer(<CursorBlink />).toJSON();
  expect(tree).toMatchSnapshot();
});
