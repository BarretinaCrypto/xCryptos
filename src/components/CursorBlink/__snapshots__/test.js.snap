// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders by default 1`] = `
<View
  collapsable={undefined}
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 0.75)",
      "height": 16,
      "marginLeft": 3,
      "opacity": 0,
      "width": 2,
    }
  }
/>
`;
